theurgy.lic

```diff
@@ -335,7 +335,7 @@ class TheurgyActions
 
   def buy_single_supply(item, shop)
     if shop['method']
-      send(shop['method'], item)
+      send(shop['method'])
```

Comment:
Was this change intentional?

Reply:
Updated my copy to the latest merged copy.

Reply:
Updated my copy to the latest merged copy.

Reply:
Updated my copy to the latest merged copy.